applications/plot_layout_array.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/python3
+
+"""
+    Summary
+    -------
+    This application plots the layout array and saves into pdf and png files.
+
+    It accepts as input the telescope list file, the name of the layout, a sequence of arguments
+    with the telescope files or a sequence of arguments with the layout names.
+
+    A rotation angle in degrees can be passed in case the array should be rotated before plotting.
+    A sequence of arguments for the rotation angle is also permitted, in which case all of them
+    are plotted and saved separately.
+
+
+    Command line arguments
+    ----------------------
+    figure_name (str, optional)
+        File name for the pdf output (without extension).
+    telescope_list (str, optional)
+        The telescopes file (.ecsv) with the array information.
+    layout_array_name (str, optional)
+        Name of the layout array.
+    rotate_angle (float, optional)
+        Angle to rotate the array before plotting (in degrees).
+    show_tel_label (bool, optional)
+        Shows the telescope labels in the plot.
+    verbosity (str, optional)
+        Log level to print (default=INFO).
+
+    Example
+    -------
+    .. code-block:: console
+
+        python applications/plot_layout_array.py --figure_name northern_array_alpha \
+        --layout_array_name test_layout
+
+"""
+
+import logging
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+from astropy import units as u
+
+import simtools.util.general as gen
+from simtools import io_handler
+from simtools.configuration import configurator
+from simtools.layout.layout_array import LayoutArray
+from simtools.visualization.visualize import plot_array
+
+
+def _parse(label, description, usage):
+    """
+    Parse command line configuration
+
+    Parameters
+    ----------
+    label: str
+        Label describing application.
+    description: str
+        Description of application.
+    usage: str
+        Example on how to use the application.
+
+    Returns
+    -------
+    CommandLineParser
+        Command line parser object
+
+    """
+    config = configurator.Configurator(label=label, description=description, usage=usage)
+
+    config.parser.add_argument(
+        "--figure_name",
+        help="Name of the output figure to be saved into as a pdf.",
+        type=str,
+        required=False,
+        default=None,
+    )
+    config.parser.add_argument(
+        "--rotate_angle",
+        help="Angle to rotate the array before plotting (in degrees).",
+        type=str,
```

Comment:
Why is this a string and not a float?

Reply:
I had some troubles with initializing it as None (and type float) so I defined it to be a str. would it be better to define it as float and initialize as 0.0?